packages/analytics/analytics-geo-map/README.md

```diff
@@ -0,0 +1,105 @@
+# @kong-ui-public/analytics-geo-map
+
+This Vue component provides a geographical map visualization based on `maplibre-gl` and displays country metrics using color-coded layers. It supports dynamic updates to country metrics and allows focusing on specific countries.
+Renders provided GeoJSON data into a choropleth style map.
+
+- [Features](#features)
+- [Requirements](#requirements)
+- [Usage](#usage)
+  - [Install](#install)
+  - [Props](#props)
+  - [Example](#example)
+
+## Features
+
+- Interactive geographical map visualization using `maplibre-gl`.
+- Color-coded countries based on provided metrics.
+- Tooltips showing country names and their corresponding metrics.
+- Ability to focus on specific countries with customizable bounds.
+- Dynamic updates to country metrics and map features.
+
+## Requirements
+
+- `vue` must be initialized in the host application.
+- `maplibre-gl` must be added as a `dependency` in the host application. [See here](https://www.npmjs.com/package/maplibre-gl)
+
+## Usage
+
+### Install
+
+First, ensure you have `maplibre-gl` installed:
+
+```bash
+npm install maplibre-gl
+npm install @kong-ui-public/analytics-geo-map
+```
+
+### Props
+
+#### `countryMetrics`
+
+- **type:** `Object`
+- **required:** `true`
+- **description:** An object mapping country codes to metric values. These metrics determine the color of each country on the map.
+
+#### `geoJsonData`
+
+- **type:** `Object`
+- **required:** `true`
+- **description:** GeoJSON data for the map, defining the shapes and positions of countries.
+
+#### `center`
+
+- **type:** `Object`
+- **required:** `false`
+- **default:** `null`
+- **description:** Initial center of the map. Expects an object with `lat` and `lng` properties.
+
+#### `fitToCountry`
+
+- **type:** `String`
+- **required:** `false`
+- **default:** `''`
+- **description:** Country code to fit the map bounds to. Overrides initial center and zoom.
+
+#### `initialZoom`
+
+- **type:** `Number`
+- **required:** `false`
+- **default:** `null`
```

Comment:
If not required, we should fall back to a certain value. A value closer to zero maybe?

Reply:
made this Number | null